.github/workflows/test.yml

```diff
@@ -6,26 +6,35 @@ on:
     inputs:
       eve_image:
         type: string
-      runner:
-        type: string
-        default: buildjet-4vcpu-ubuntu-2204
   workflow_call:
     inputs:
       eve_image:
         type: string
-      runner:
-        type: string
-        default: buildjet-4vcpu-ubuntu-2204
 
 jobs:
+  determine-runner:
+    name: Determine best available runner
+    runs-on: ubuntu-latest
+    outputs:
+      runner: ${{ steps.fork-check.outputs.runner }}
+    steps:
+      - id: fork-check
+        run: |
+          if ["${{ github.event.repository.full_name}}" == "lf-edge/eve" || "${{ github.event.repository.full_name}}" == "lf-edge/eden"]; then
```

Comment:
Would it be possible to extend this check down to `action/setup-environment` and enable `eve.accel` if workflow is running on buildjet?

Reply:
I thought we could actually change setup-environment and use some utilities to determine if hw acceleration is possible, that way we don't have coupling with runner, but rather with capability

Reply:
Something like this: https://manpages.ubuntu.com/manpages/impish/man1/kvm-ok.1.html ?

Reply:
Yes, I also found option with lscpu :), sending it rn

Reply:
I am not sure that current logic works. I can see SandyBridge [here](https://github.com/lf-edge/eden/actions/runs/6642450932/job/18047461117?pr=909#step:3:1476), but it should be [host](https://github.com/lf-edge/eden/blob/a0dbf14c57ec8f9c7897ea9bbb569a800bc23462/pkg/defaults/defaults.go#L126) in case of acceleration enabled.